parts/k8s/kubernetesagentresourcesvmas.t

```diff
@@ -320,7 +320,7 @@
         {{if IsOpenShift }}
           "script": "{{ Base64 (OpenShiftGetNodeSh .) }}"
         {{else}}
-          "commandToExecute": "[concat('for i in $(seq 1 1200); do if [ -f /opt/azure/containers/provision.sh ]; then break; fi; if [ $i -eq 1200 ]; then exit 100; else sleep 1; fi; done; ', variables('provisionScriptParametersCommon'),' /usr/bin/nohup /bin/bash -c \"/bin/bash /opt/azure/containers/provision.sh >> /var/log/azure/cluster-provision.log 2>&1\"')]"
+          "commandToExecute": "[concat('for i in $(seq 1 1200); do if [ -f /opt/azure/containers/provision.sh ]; then break; fi; if [ $i -eq 1200 ]; then exit 100; else sleep 1; fi; done; ', variables('provisionScriptParametersCommon'),' {{if IsNSeriesSKU .}}GPU_NODE=true {{end}}/usr/bin/nohup /bin/bash -c \"/bin/bash /opt/azure/containers/provision.sh >> /var/log/azure/cluster-provision.log 2>&1\"')]"
```

Comment:
why is `{{if IsNSeriesSKU .}}GPU_NODE=true {{end}}` not part of 'provisionScriptParametersCommon'

Reply:
Because that doesn't have the particular agent pool context that we need (to determine "am I a GPU node?")

Reply:
Gotcha

Reply:
Small preference for: ```GPU_NODE={{IsNSeriesSKU .}}``` but otherwise 👍